packages/components/app/styles/components/pagination.scss

```diff
@@ -166,3 +166,25 @@ $pagination-height: calc(36px + 6px);
     background-position: center right 5px;
   }
 }
+
+// Responsive layout
+// max-width is based on where component layout breaks down
+@media screen and (max-width: 1000px) {
+  .hds-pagination {
+    display: flex !important;
+    flex-direction: column;
+    height: auto !important;
+
+    > * {
```

Comment:
we should avoid using these kind of generic selectors and prefer to be more declarative/specific on what we're targeting.

Reply:
Ok but this is targeting flex children. I will change it.

Reply:
I know but it's not an indeterminate or unknown list of children. we know which ones are the children we want to target